cmd/core/main.go

```diff
@@ -138,13 +141,23 @@ func main() {
 		os.Exit(1)
 	}
 
+	var capabilities []string
+	switch enableCaps {
+	case "none":
```

Comment:
how do users know those values?

Reply:
They don't know, I will document the values.

Approximately, they are the `TYPE` from `vela workloads/traits`.
```
TYPE       	CATEGORY	DESCRIPTION
+task      	workload	One-off task to run a piece of code or script to completion
+webservice	workload	Long-running scalable service with stable endpoint to receive external traffic
+worker    	workload	Long-running scalable backend worker without network endpoint
+autoscale 	trait   	Automatically scale the app following certain triggers or metrics
+metrics   	trait   	Configure metrics targets to be monitored for the app
+rollout   	trait   	Configure canary deployment strategy to release the app
+route     	trait   	Configure route policy to the app
+scaler    	trait   	Manually scale the app
```

But in fact we only support "metrics", "route", "autoscale" as others don't have backend controllers. Besides, the values include "podspecworkload".

Reply:
It's not friendly for platform builders. Fortunately, most of time, just setting the flag to `all` is enough to make Vela compatible with previous OAM Kubernetes Runtime.